.release-it.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+const releaseItConfig = {
+    verbose: false,
+    force: false,
+    isDryRun: false,
+    git: {
+        changelog: 'git log --pretty=format:"* %s (%h)" ${latestTag}...HEAD',
+        requireCleanWorkingDir: true,
+        addUntrackedFiles: false,
+        requireBranch: false,
+        requireUpstream: true,
+        requireCommits: false,
+        commit: true,
+        commitMessage: "chore(release): release ${version}",
+        commitArgs: "",
+        tag: true,
+        tagName: "${version}",
+        tagAnnotation: "Release",
```

Comment:
I prefer to use a "v" prefix for releases, for example "v1.0.0" and "v2.5.2". Can this script be updated to use that convention?

Reply:
Yep, I guess so. I'll check it out and update the PR

Reply:
Should be okay now!